desktop/models/partner_show.coffee

```diff
@@ -321,6 +321,9 @@ module.exports = class PartnerShow extends Backbone.Model
   isFairBooth: ->
     @has 'fair'
 
+  isOnlineExclusive: ->
+    not @has('location') && not @has('fair_location') && not @has('partner_location')
```

Comment:
urgh, me no likey coffeescript `and` `or` and `not`.

Maybe its better to just use `!` instead of `not`? I know its not 'idiomatic' CS, but, ugh, coffeescript is silly.

Reply:
yeah i have no idea, i just looked at few examples in this file, happy to switch to `!`, def more readable than `not @has`

Reply:
not sure why this build start braking! looks like some random unrelated failure

Reply:
This is different from Metaphysics' logic, both in that it requires a fair location and ignores `partner_city`: https://github.com/artsy/metaphysics/pull/927/files#diff-787a1e14fd2b358d2ab504851b2d878bR189

Reply:
good catch, i fixed this on the MP side and missed this, fixing now!